_dashboards/visualize/cluster-layer.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Maps cluster layer
+nav_order: 50
+grand_parent: Building data visualizations
+parent: Using coordinate and region maps 
+has_children: false
+---
+
+# Maps cluster layer
+Introduced 2.9
+{: .label .label-purple }
+
+The cluster layer aggregates all points and shapes from an index using geospatial aggregations. Examples of how you can use the maps cluster layer include: 
+
+- Visualizing the distribution of your data
+- Tracking the movement of objects
+- Identifying clusters of data
+- Creating heatmaps
+
+## Prerequisites
+
+To use the maps cluster layer in OpenSearch:
+
+- <what are the prerequistes?>
+- <do we need to reference the Maps Stats API?>
+
+To enable the cluster layer:
```

Comment:
Need testing endpoint.

Reply:
@VijayanB Will you send me the demo and endpoint for testing this feature? I need to wrap up the draft documentation tomorrow. I'll then send you the draft content for your technical review. Thanks!